server/views/pages/appointments/create-and-edit/apply-to.njk

```diff
@@ -9,7 +9,11 @@
 {% set editMessage = getAppointmentEditMessage(session.appointmentJourney, session.editAppointmentJourney) %}
 {% set pageTitle = appointmentJourneyTitle(editMessage + ' which appointments', session.appointmentJourney) %}
 {% set pageId = 'appointments-manage-apply-to-page' %}
-{% set backLinkHref = ('../' + property if property else "/appointments/" + appointmentId + "/occurrence/" + occurrenceId) %}
+{% if property == 'date-and-time' %}
```

Comment:
I think this needs ` || editAppointmentJourney.addPrisoners` as that's the other path that would return to the schedule view